src/app/config/projectsJsonData.ts

```diff
@@ -77,4 +77,20 @@ export default class projectsJsonData{
       }
       return undefined;
     }
+
+    getProjectTemplateRepository(projectTypeKey: string, scriptType: string)
+    {
+      for (let key in this.m_projectJsonData.projectTypes)
+      {
+        if (_.toLower(projectTypeKey) == key){
+          if (scriptType == 'javascript'){
```

Comment:
Why not just use scriptType as a key in:

`return this.m_projectJsonData.projectTypes[key].templates[scriptType].repository; `

Reply:
Duh. Good call :-)